src/test/java/org/rulez/demokracia/PDEngine/VoteCreationTest.java

```diff
@@ -110,4 +111,340 @@ public void create_creates_a_vote_with_adminKey() {
 		assertEquals(VoteRegistry.getByKey(adminInfo.adminKey).adminKey, adminInfo.adminKey);
 	}
 
+// voteName validation tests
+
+	@Test
+	public void votename_contains_wrong_characters_input1() throws ReportedException {
+		List<String> neededAssurances = new ArrayList<>();
```

Comment:
but it is okay for a vote to contain space.
so use vote_name_can_contain_space and reverse the test